lib/timers.js

```diff
@@ -169,7 +169,11 @@ function listOnTimeout() {
     // Check if this loop iteration is too early for the next timer.
     // This happens if there are more timers scheduled for later in the list.
     if (diff < msecs) {
-      this.start(msecs - diff, 0);
+      var timeRemaining = msecs - (TimerWrap.now() - timer._idleStart);
```

Comment:
I thought about it but since this is now that the code fixing this issue is not on its own, it didn't seem like a decent place to do it.  But we could mention the issue(s) where this block is addressing bugs.